gadget-container/entrypoint.sh

```diff
@@ -164,8 +164,29 @@ if [ "$TOOLS_MODE" = "auto" ] || [ -z "$TOOLS_MODE" ] ; then
     echo "BTF is available: Using CO-RE based tools"
     TOOLS_MODE="core"
   else
-    echo "BTF is not available: Using standard tools"
-    TOOLS_MODE="standard"
+    echo "BTF is not available: Trying btfhub"
+    source /etc/lsb-release
+
+    DISTRO=$(echo $DISTRIB_ID | tr '[:upper:]' '[:lower:]')
+    KERNEL=$(uname -r)
+    URL=https://github.com/aquasecurity/btfhub/raw/main/$DISTRO/$DISTRIB_RELEASE/$KERNEL.btf.tar.xz
```

Comment:
Since this is running in the gadget container based on Ubuntu, DISTRO is always going to be `ubuntu` and DISTRIB_RELEASE is always `20.04`. 

In line 69, there is an example how to get the distro on the host, using the `/host` directory:

```
if grep -q '^ID="rhcos"$' /host/etc/os-release > /dev/null ; then
```

Reply:
oh right. I wasn't a good idea to test it on a host running the same distro of the container. I updated it, thanks!